src/bin/common/solving_loop.ml

```diff
@@ -357,9 +357,9 @@ let main () =
          PR. See https://github.com/OCamlPro/alt-ergo/pull/553 *)
       if Stdlib.(Options.get_sat_solver () = Tableaux) then
         Options.set_interpretation ILast
-      else Printer.print_wrn "%a The generation of models is not supported \
-                              for the SAT solver %a. Please choose the \
-                              SAT solver Tableaux."
+      else Printer.print_wrn "%a The generation of models is not supported yet \
+                              by the SAT solver %a. Please use the \
+                              meta-option `--produce-models`."
```

Comment:
What is meta about that option?

Reply:
I thought this option turns some other options on. I can remove it.

Reply:
Ah, right -- it is roughly equivalent to `--sat-solver Tableaux --interpretation last` indeed (but not exactly; `--interpretation` can still be overwritten separately) but that is an implementation detail, from the user point of view it is just an option.